react/Footer/Footer.js

```diff
@@ -64,6 +64,9 @@ export default class Footer extends Component {
             <div className={styles.columns}>
               <FooterNav label="Tools">
                 { tools.map(this.renderLink) }
+                <ToggleContainer name="DownloadAppsToggle" label="Download Apps">
```

Comment:
Should the label be `Download apps`?

Reply:
Thats the label Socrates requested they be under, happy to change it if you have a more appropriate name though

Reply:
Oh never mind. Just saw what you meant 🤦‍♂️